package/gluon-luci-wifi-config/files/usr/lib/lua/luci/model/cbi/admin/wifi-config.lua

```diff
@@ -0,0 +1,90 @@
+local f, s, o, p, q
```

Comment:
The variables `p` and `q` aren't used at top level, but only in a lower scope (inside a loop), so declare the variables at the scope they are actually used.